security/webauthn/src/test/java/MySqlWebAuthnIT.java

```diff
@@ -0,0 +1,32 @@
+import io.quarkus.test.bootstrap.MySqlService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MySqlWebAuthnIT extends AbstractWebAuthnTest {
+
+    private static final String MYSQL_USER = "quarkus_test";
+    private static final String MYSQL_PASSWORD = "quarkus_test";
+    private static final String MYSQL_DATABASE = "quarkus_test";
+    private static final int MYSQL_PORT = 3306;
+
+    @Container(image = "${mysql.80.image}", port = MYSQL_PORT, expectedLog = "ready for connections")
+    static MySqlService database = new MySqlService()
+            .withUser(MYSQL_USER)
+            .withPassword(MYSQL_PASSWORD)
+            .withDatabase(MYSQL_DATABASE);
```

Comment:
Can you name one thing that setting `withUser`, `withPassword` and `withDatabase` will give us? I don't see an issue with it, but it's unnecessary noise, I'd only set there is some positive output of setting it. I tried it without these 3 methods and works.

Reply:
You are right, the app is using the default config from the container's image config so here is not needed, I am removing those custom configurations.